README.md

```diff
@@ -172,6 +172,19 @@ OPTIONS:
 EXAMPLE: run.sh -s "Squeak-trunk" --headfull /path/to/projects
 ```
 
+## Running locally
+
+If you need to run smalltalkCI locally you can follow these steps:
+
+1. clone your repository
+  * a) from github: `git clone https://github.com/your/project.git`, or
+  * b) clone your local repo: `git clone /wherever/is/your/project`, or
+  * c) use your local repo directly
+2. cd into the cloned directory `cd project`
+3. download smalltalkCI: `wget -O smalltalkCI.zip https://github.com/hpi-swa/smalltalkCI/archive/master.zip`
+4. unzip it: `unzip -o smalltalkCI.zip`
+5. run smalltalkCI, provide dialect you want to test and your local config: `smalltalkCI-master/run.sh --headfull -s "Pharo-5.0" .smalltalk.ston`
+
```

Comment:
I understand, but part of the reason for documenting this independently (in my opinion) is that smalltalkCI can be used as CI for local projects (obviating the need to download Jenkins to run automated tests) and in this scenario you might want an independent git checkout of smalltalkCI so that you smalltalkCI setup can be used to run tests against a variety of projects...

in the end I think it's worth covering both scenarios ... 

Perhaps a link to the reproduce travis-problem scenario can be included in the ` to reproduce bug` message ... There will be additional options for launching builds for GemStone (see #52) and this will turn out to be a tidy spot for documenting those options when they apear